try-it.md

```diff
@@ -147,17 +83,15 @@ downloaded from the `IMAGE_LOCATION` value configured.
 
 > warning "Warning"
 > If you see this error during the installation:
->
-> ```sh
-> error: failed to connect to the hypervisor
+>```
+> error: failed to connect to the hypervisor \
 > error: Failed to connect socket to '/var/run/libvirt/libvirt-sock':  Permission denied
-> ```
->
+>```
 > You may need to log out then login again, and run `make clean` and `make` again.
 
-## 2. Working with the Environment
+## 2. Working with Environment
 
-### 2.1. Bare Metal Hosts
+### 2.1. BareMetalHosts
```

Comment:
I think according to our style guide we want this to stay "Bare Metal Hosts"?

Reply:
For some reason I thought we are using BareMetalHosts anywhere. But apparently I was wrong.
Fixed, thank you.